SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -1,137 +1,332 @@
+                                           CreateNoWindow = true
+                                       }
+                       };
+            Console.WriteLine(proc.StartInfo.Arguments);
+
+            proc.Start();
+            proc.WaitForExit();
+
+            return proc.StandardOutput.ReadToEnd();
+        }
 
-        public Hooks(IObjectContainer objectContainer)
+        /// <summary>
+        /// Setups the security service container.
+        /// </summary>
+        /// <param name="containerName">Name of the container.</param>
+        /// <param name="logger">The logger.</param>
+        /// <param name="imageName">Name of the image.</param>
+        /// <param name="networkService">The network service.</param>
+        /// <param name="hostFolder">The host folder.</param>
+        /// <param name="dockerPort">The docker port.</param>
+        /// <param name="dockerCredentials">The docker credentials.</param>
+        /// <param name="forceLatestImage">if set to <c>true</c> [force latest image].</param>
+        /// <param name="additionalEnvironmentVariables">The additional environment variables.</param>
+        /// <returns></returns>
+        private static IContainerService SetupSecurityServiceContainer(String containerName,
+                                                                       ILogger logger,
+                                                                       String imageName,
+                                                                       INetworkService networkService,
+                                                                       String hostFolder,
+                                                                       Int32 dockerPort,
+                                                                       (String URL, String UserName, String Password)? dockerCredentials,
+                                                                       Boolean forceLatestImage = false,
+                                                                       List<String> additionalEnvironmentVariables = null)
         {
-            this.ObjectContainer = objectContainer;
+            logger.LogInformation("About to Start Security Container");
+
+            List<String> environmentVariables = new List<String>();
+            environmentVariables.Add($"ServiceOptions:PublicOrigin=http://{containerName}:{dockerPort}");
+            environmentVariables.Add($"ServiceOptions:IssuerUrl=http://{containerName}:{dockerPort}");
+            environmentVariables.Add("ASPNETCORE_ENVIRONMENT=IntegrationTest");
+            environmentVariables.Add("urls=http://*:5551");
+
+            if (additionalEnvironmentVariables != null)
+            {
+                environmentVariables.AddRange(additionalEnvironmentVariables);
+            }
+
+            String containerFolder = FdOs.IsLinux() ? "/home/txnproc/trace" : "C:\\home\\txnproc\\trace";
```

Comment:
Codacy found an issue: [Remove the unused local variable 'containerFolder'.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)